install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.yaml

```diff
@@ -304,3 +304,12 @@ admin:
   address:
     pipe:
       path: "/var/run/cilium/envoy/sockets/admin.sock"
+layered_runtime:
+  layers:
+  - name: static_layer
+    {{- with .Values.envoy.runtime }}
+    static_layer:
```

Comment:
IIRC, there might be multiple layers, wdyt about mapping .Values.enovy.runtime -> layers_runtime directly?

Reply:
My idea of adding only a `static_layer` was to simplify the configuration in the Helm values, as I'm unsure if using the other layer types would be beneficial here. However, I don't have strong opinions. Please let me know, and I can modify it to allow a list of `config.bootstrap.v3.RuntimeLayer` items to make it more flexible.